src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars--0">
```

Comment:
Add block name to classes and this advice will work and using a block modifier without the block itself is incorrect according to BEM)
Add this class to each divs with modifier
![image](https://github.com/mate-academy/layout_stars/assets/70009205/8f684776-1d88-4153-bea3-26fe87c1cc85)

```suggestion
    <div class="stars stars--0">
```